OpenRA.Game/Graphics/SpriteLoader.cs

```diff
@@ -21,6 +22,12 @@ public interface ISpriteLoader
 		bool TryParseSprite(Stream s, out ISpriteFrame[] frames);
 	}
 
+	public interface ISpriteLoader2 : ISpriteLoader
```

Comment:
For the purposes of this PR we can (as a prerequisite commit) add a separate commit that passes the sprite filename as an extra argument to `ISpriteLoader.TryParseSprite`.  It's a bit of a design wart that we haven't done this before now.